doc/design.md

```diff
@@ -281,6 +289,91 @@ Covers:
 
 Needs: impl, itest
 
+### ExaOperation Emulation
+
+EXAoperation features required for integration tests are emulated.
+
+Interface compatibility for plug-ins means that the emulation uses the established plug-in interfaces (read "control scripts inside the plug-in").
+
+EXAoperations own interfaces are _not_ available. Especially there is no emulation of the XML-RPC interface. What is available though is a control object implementation a Java interface called "EXAoperation". It is this interface that test authors can use in test code to trigger the emulated features.
+
+#### Extracting Plug-in Packages
+`dsn~extracting-plug-in-packages~1`
+
+The `ExaOperationEmulator` unpacks plug-in packages from TAR archives with the suffix `.pgk` inside the Exasol instance.
+
+Covers:
+
+* `req~installing-an-exaoperation-plug-in~1`
+
+Needs: impl, itest
+
+#### Installing Plug-ins
+`dsn~installing-plug-ins~1`
+
+The `Plugin` calls the script "`install`" from the extracted plug-in contents to complete the installation process.
```

Comment:
The only required script within a plugin is the one called "plugin-functions". Anything else is plugin-specific.
That script, however, should provide the "INSTALL" function.

Reply:
I see. I will implement this and additionally make `runPluginScript()` public.